package.json

```diff
@@ -62,7 +62,7 @@
     "better-than-before": "1.0.0",
     "chai": "3.4.1",
     "chokidar-cli": "1.2.0",
-    "conventional-changelog-core": "1.0.1",
+    "conventional-changelog-core": "1.8.0",
```

Comment:
Thanks Nick. Give me a few hours and I'll checkout the code, run it and see how it goes. I'm puzzled by why I made "conventional-changelog-core" a devDependency, and I want to ensure it isn't actually a runtime dependency.